tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchCMISTests.java

```diff
@@ -165,6 +169,15 @@ public void matchDocumentName()
         searchQueryService.expectResultsFromQuery(query, user1, FILE_0_NAME);
     }
 
+    @TestRail (description = "Check ORDER BY syntax works as expected", section = TestGroup.SEARCH, executionType = ExecutionType.REGRESSION)
+    @Test (groups = TestGroup.SEARCH)
+    public void checkOrderByAscSyntax()
+    {
+        List<String> expectedList = orderNames(FILE_0_NAME, FILE_1_NAME, FILE_2_NAME);
```

Comment:
Could we also have a test for using `ORDER BY cmis:name DESC`?

Reply:
@gevagel Mentioned in slack

>  Tests for reverse sort are already in place.

Thanks. Can you add comment here (with link to code / commit) to confirm we test sort in both directions.

I'm sure Tom or you can then resolve this conversation.